docs/source/configuration_file.md

```diff
@@ -127,6 +127,13 @@ Uncertainty computation is performed if `n_it>0` and at least `epistemic` or `al
 - `aleatoric`: Bool. Image-based uncertainty with [test-time augmentation](https://doi.org/10.1016/j.neucom.2019.01.103).
 - `n_it`: Integer. Number of Monte Carlo iterations. Set to 0 for no uncertainty computation.
 
+## Cascaded Architecture Features
+
+### object_detection_params (Optional)
+- `object_detection_path`: String. Path to object detection model. The model's prediction will be used to generate bounding boxes.
```

Comment:
Is it the path to the model? I thought it was the path to the predictions of the first step (.json).

Reply:
No it's the path to the detection model to apply segmen_volume. The .json has always the same path: log_directory/bounding_box.json